modules/dlscript/godot.cpp

```diff
@@ -180,6 +181,16 @@ void GDAPI *godot_dlinstance_get_userdata(godot_object *p_instance) {
 	return NULL;
 }
 
+void GDAPI godot_get_global_constants(godot_dictionary *p_constants) {
```

Comment:
My point was to make more explicit memory management should be done on user's side.
I've updated the function signature to `godot_dictionary GDAPI godot_get_global_constants()` but I'm a bit concern calling `godot_dictionary_clear(&ret_value)` to free dictionary is counter-intuitive...